editor.planx.uk/src/pages/FlowEditor/components/Settings/DesignSettings/FaviconForm.tsx

```diff
@@ -39,8 +60,16 @@ export const FaviconForm: React.FC = () => {
       input={
         <InputRow>
           <InputRowLabel>Favicon:</InputRowLabel>
-          <InputRowItem width={50}>
-            <PublicFileUploadButton />
+          <InputRowItem width={formik.values.favicon ? 90 : 50}>
+            <ImgInput 
```

Comment:
I've updated this to use the wrapper `ImgInput` component which gives us a preview and a way to remove the image.

I'll open up a very small follow-on PR to make this change to the Logo form shortly to keep this visually and behaviorally consistent.